src/libraries/System.Private.CoreLib/src/System/Threading/WaitHandle.cs

```diff
@@ -111,21 +116,74 @@ private bool WaitOneNoCheck(int millisecondsTimeout)
             SafeWaitHandle? waitHandle = _waitHandle;
             ObjectDisposedException.ThrowIf(waitHandle is null, this);
 
+            bool isWaitHandleKeywordEnabled = NativeRuntimeEventSource.Log.IsEnabled(
+                EventLevel.Informational,
+                NativeRuntimeEventSource.Keywords.WaitHandleKeyword);
+            if (isWaitHandleKeywordEnabled)
+            {
+                NativeRuntimeEventSource.Log.WaitHandleWaitStart(
+                    NativeRuntimeEventSource.WaitHandleWaitSourceMap.Unknown,
+                    0);
+            }
+
```

Comment:
This can be deleted, it's left over from before and is replaced by the changes in my commit